Neos.Flow/Classes/Validation/Validator/GenericObjectValidator.php

```diff
@@ -43,6 +48,15 @@ public function setValidatedInstancesContainer(\SplObjectStorage $validatedInsta
         $this->validatedInstancesContainer = $validatedInstancesContainer;
     }
 
+    /**
+     * @param object $object The object instance to register a validation callback for
+     * @param \Closure $callback The callback to invoke once this object has been validated
+     */
+    public static function onValidated(object $object, \Closure $callback)
+    {
+        self::$validationCallbacks[spl_object_hash($object)][] = $callback;
```

Comment:
With 6.0 (PHP 7.2+) we could replace this with `spl_object_id()`, which is a tiny bit faster and has same semantics.

Reply:
Go for it!

Reply:
```suggestion
        self::$validationCallbacks[spl_object_id($object)][] = $callback;
```